pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -187,7 +188,9 @@ func (w *Watcher) traversePluginDir(dir string) error {
 				return fmt.Errorf("failed to watch %s, err: %v", path, err)
 			}
 		case mode&os.ModeSocket != 0:
+            w.wg.Add(1)
 			go func() {
+                defer w.wg.Done()
```

Comment:
Is it me or the intendation looks out of place?

Reply:
not only you, the indent is indeed off.